nixos/modules/services/network-filesystems/glusterfs.nix

```diff
@@ -59,9 +59,9 @@ in
         default = null;
         type = types.nullOr (types.submodule {
           options = {
-            tlsKey = mkOption {
+            tlsKeyPath = mkOption {
               default = null;
-              type = types.path;
+              type = types.str;
```

Comment:
This does not protect against accidentally copying things into the store. It still accepts path literals, as in `/my/secret.key` which will be copied on eval.